README.md

```diff
@@ -1,2 +1,50 @@
 # terraform-examples
-Deploy Every Possible Flagsmith Adventure Terraform
+Terraform scripts for example deployments to popular cloud providers.
+
+## EKS
+
+The following prerequisites are expected to apply this example configuration successfully:
+
+- Terraform: [install instructions](https://developer.hashicorp.com/terraform/downloads).
+- An up and running EKS cluster with a VPC set up for at least two availability zones. [Read an official example on Terraform EKS provisioning](https://developer.hashicorp.com/terraform/tutorials/kubernetes/eks).
```

Comment:
Couple of questions here: 

 1. I'm assuming the need to have the cluster set up already is because we want to set up the 'internals' of the cluster and need the cluster to exist already to do so? I'm sure I came across this previously with ECS too. Are there no ways around this still? The ideal (I think?) would be to define the cluster in the terraform example and reference it in the rest of the resources. 
 2. Is there a reason that we specify 'at least two availability zones' ? Obviously it's best practice to do so, but is there a technical reason we need at least 2?

Reply:
1. We can vendor https://github.com/hashicorp/learn-terraform-provision-eks-cluster but I deemed this redundant. I'm down for doing this if we figure out how make this optional – I believe it's useful to have an ability to deploy to a pre-existing cluster without the user having to figure out what to modify/comment out.
2. Yes, it's an RDS prerequisite. I've included a link to RDS networking docs in the  'Resources' section.

Reply:
Added a pointer on a dedicated EKS cluster provisioning to README.md.